src/app/clusters/thread-border-router-management-server/thread-border-router-management-server.cpp

```diff
@@ -264,6 +280,12 @@ CHIP_ERROR ServerInstance::Read(const ConcreteReadAttributePath & aPath, Attribu
                                                    : aEncoder.EncodeNull();
         break;
     }
+    case Attributes::PendingDatasetTimestamp::Id: {
+        Optional<uint64_t> pendingDatasetTimestamp = ReadPendingDatasetTimestamp();
+        status = pendingDatasetTimestamp.HasValue() ? aEncoder.Encode(DataModel::MakeNullable(pendingDatasetTimestamp.Value()))
```

Comment:
```suggestion
        status = pendingDatasetTimestamp.HasValue() ? aEncoder.Encode(pendingDatasetTimestamp.Value())
```
There's no need to wrap the value back into a Nullable for `Encode` since you have already checked for null

Reply:
Done